src/components/common/SwapPrice.stories.tsx

```diff
@@ -1,23 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Meta, Story } from '@storybook/react/types-6-0'
 import { SwapPrice, Props } from './SwapPrice'
 import { GNO, DAI } from 'storybook/tokens'
+import { CenteredAndFramed } from 'storybook/decorators'
 
 export default {
   title: 'Common/SwapPrice',
+  component: SwapPrice,
+  decorators: [CenteredAndFramed],
 } as Meta
 
-const Template: Story<Props> = (props) => <SwapPrice {...props} />
+const Template: Story<Partial<Props>> = (props) => {
+  const [isPriceInverted, setIsPriceInverted] = useState(false)
 
-export const Basic = Template.bind({})
-Basic.args = {
-  baseToken: GNO,
-  quoteToken: DAI,
-  isPriceInverted: true,
+  return (
+    <SwapPrice
+      baseToken={GNO}
+      quoteToken={DAI}
+      isPriceInverted={isPriceInverted}
+      onSwapPrices={(): void => {
```

Comment:
is anything supposed to happen onClick? I see actions logged in the Actions tab. But no console.logs, not even inside iframe context

Reply:
Yes, that's an issue. It happens because https://github.com/gnosis/dex-react/blob/develop/.storybook/preview.tsx#L7 intercepts the call overriding me!

I'm not sure I fully understand actions here, so probably missing the point why this was added in the first place, but I would say actions can be defined locally to the specific story if they are needed

I'll remove it in this PR, but let me know your thoughts

Reply:
I think is better to keep actions local instead of having the global one. 
This way, we are more declarative with the properties, and we don't override the `on<Something>` automatically.

In my case I believe it was overridden, cause I allow to provide as parameters a Partial of the Props, so I can define the defaults, and let anyone use the template to override any prop. In this case, I'm being injected the `onSwapPrices` automatically because of the naming convention